u3-views/views/partials/video-card.hbs

```diff
@@ -0,0 +1,8 @@
+<a class='home-card' href='/videos/{{tease.id}}'>
+  <div>
+    <img src='/images/{{tease.id}}.webp' />
+    <p>{{card.duration}}</p>
+  </div>
+  <p>{{card.title}}</p>
+  <p class='stats'>{{card.viewsShort}} views · {{card.dateRelative}}</p>
+</a>
```

Comment:
คุณกำลังผสม `tease` และ `card` เลือกหนึ่ง! ฉันแนะนำ `card` (เนื่องจากวัตถุนี้เป็นข้อมูลสำหรับการ์ดวิดีโอ 1 การ์ด)
You are mixing `tease` and `card`. Pick one! I suggest `card` (since this object is the data for 1 video-card).

Reply:
(I fixed it for you)